purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesOrchestrator.kt

```diff
@@ -1299,7 +1299,7 @@ internal class PurchasesOrchestrator(
                 )
 
                 // Synchronize paywall events after a new purchase
-                flushPaywallEvents()
+                flushPaywallEvents(Delay.NONE)
```

Comment:
I think it makes sense to have no delay when flushing the events after a purchase, but wanted to make sure, since a comment above suggests that `Delay.NONE` was only added to flushes when the app got backgrounded.